src/sympc/tensor/replicatedshare_tensor.py

```diff
@@ -695,9 +716,8 @@ def method_new_rs_tensor(
             _self: "ReplicatedSharedTensor", *args: List[Any], **kwargs: Dict[Any, Any]
         ) -> Any:
             shares = []
-            for i in range(len(_self.shares)):
-                tensor = getattr(_self.shares[i], method_name)(*args, **kwargs)
```

Comment:
Q: Could we leave them as they were before since it might help  when we try to debug

Reply:
Makes sense.